setup/FSharp.SDK/component-groups/Compiler_Redist.wxs

```diff
@@ -124,30 +124,6 @@
         </RegistryKey>
       </Component>
 
-      <Component Id="Compiler_Redist_Microsoft.Build.Conversion.Core.dll" Guid="$(fsharp.guid(Compiler_Redist_Microsoft.Build.Conversion.Core.dll, $(var.LocaleCode)))">
```

Comment:
I don't think we can get rid of this any time soon.  It is because the deskto copiler has a dependency on some msbuild dlls.  The latest version of those dll's require that certain Apis can only be called from within msbuild.exe.

So .. changing these will likely result in fsi.exe not working correctly most of the time, and command line FSC builds not working some of the time.